bucket/detect-it-easy.json

```diff
@@ -0,0 +1,40 @@
+{
+    "version": "3.03",
+    "description": "Program for determining types of files",
+    "homepage": "http://ntinfo.biz",
+    "license": "MIT",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/horsicq/DIE-engine/releases/download/3.03/die_win64_portable_3.03.zip",
+            "hash": "f793c8ea4578cafca7543fe407f03ac50ac15850e99983f1cc5ee19b0dda7c78"
+        },
+        "32bit": {
+            "url": "https://github.com/horsicq/DIE-engine/releases/download/3.03/die_win32_portable_3.03.zip",
+            "hash": "0ce1787066ae2bad2d160aa76a2bb0f188b978b099643b15c8b0c7b58594cc1b"
+        }
+    },
+    "bin": [
+        "die.exe",
+        "diec.exe"
```

Comment:
What is the difference between these two binaries? Do both support arguments?

Reply:
I think only `diec.exe` accepts arguments.
![](https://github.com/horsicq/Detect-It-Easy/raw/master/docs/4.png)

Reply:
I think you accidentally added `die.exe` back to bin

Reply:
Oops.
fixed by https://github.com/ScoopInstaller/Extras/commit/93ac9084f093363f64db8a997afc35e23b98d679